app/src/main/java/com/example/dashcarr/presentation/tabs/settings/ConfigureRecordingFragment.kt

```diff
@@ -17,9 +18,14 @@ class ConfigureRecordingFragment : BaseFragment<FragmentConfigureRecordingBindin
 ) {
     private lateinit var bottomNavigationView: BottomNavigationView
     private val viewModel: ConfigureRecordingViewModel by viewModels()
+    private val settingsViewModel: SettingsViewModel by viewModels()
 
     private var startTimeMillis: Long = 0
 
+    var isFilteredChecked = false
+    var isAccelChecked = false
+    var isGyroChecked = false
+
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
```

Comment:
Your IDE is highlighting some stuff yellow. It is most of the time very useful to implement it to get cleaner code. Here as well you don't have to check for null twice